.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -194,19 +187,13 @@ jobs:
                   name: changelog trunk
                   path: trunk
 
-            - name: Commit the content changes
-              working-directory: ./trunk
-              run: |
-                  svn st | grep '^?' | awk '{print $2}' | xargs -r svn add
-                  svn st | grep '^!' | awk '{print $2}' | xargs -r svn rm
-                  svn commit -m "Committing version $VERSION" \
-                   --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
-
-            - name: Create the SVN tag
-              working-directory: ./trunk
-              run: |
-                  svn copy "$PLUGIN_REPO_URL/trunk" "$PLUGIN_REPO_URL/tags/$VERSION" -m "Tagging version $VERSION" \
-                   --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+            - name: Deploy WordPress Plugin
+              uses: 10up/action-wordpress-plugin-deploy@stable
+              with:
+                  dry-run: true
+              env:
+                  SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+                  SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
 
             - name: Update the plugin's stable version
               working-directory: ./trunk
```

Comment:
We might be able to use the companion https://github.com/10up/action-wordpress-plugin-asset-update here.